homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -41,8 +42,8 @@
 CONF_CODE_ARM_REQUIRED = "code_arm_required"
 
 DEFAULT_ALARM_NAME = "HA Alarm"
-DEFAULT_DELAY_TIME = datetime.timedelta(seconds=0)
-DEFAULT_PENDING_TIME = datetime.timedelta(seconds=60)
+DEFAULT_DELAY_TIME = datetime.timedelta(seconds=60)
```

Comment:
Should the default be the same as before?

Reply:
I actually changed the time the alarm stays in the "pending" state.
Before it used to be: delay_time + pending_time   (pending_time is now called arming_time)
Now it is only: delay_time

Therefore before the default time was 60 seconds: delay_time + pending_time = 0 + 60 = 60
Therefore I am now setting the delay_time to a default of 60 because that will cause the same default behaviour